docs/section5/Rule5-31.md

```diff
@@ -0,0 +1,46 @@
+
+# Envelope - Rule 5-31  
+
+**Rule ID:** 5-31  
+**Rule Description:** Manual fenestration shading devices, such as blinds or shades, shall be modeled or not modeled the same as in the baseline building design.  
+**Rule Assertion:** B-RMR subsurface.has_manual_interior_shades=P-RMR subsurface.has_manual_interior_shades  
+**Appendix G Section:** Section G3.1-5(a)(4) Building Modeling Requirements for the Proposed design and G3.1-5(d) Building Modeling Requirements for Baseline building  
+**Appendix G Section Reference:**  None
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Checks:** None  
+**Evaluation Context:**  Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  1. match_data_element()
+  2. get_opaque_surface_type()
+
```

Comment:
I don't think we need this function here, because the manual shading must be the same in the baseline and proposed on all subsurfaces irrespective of subsurface type.

Reply:
Wall check and function are removed.